LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -436,17 +465,17 @@ public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaReques
                 if (confirmDownlinkMessageBuilderResp.DownlinkPktFwdMessage != null)
                 {
                     this.receiveWindowHits?.Add(1, KeyValuePair.Create(MetricRegistry.ReceiveWindowTagName, (object)confirmDownlinkMessageBuilderResp.ReceiveWindow));
-                    _ = request.PacketForwarder.SendDownstreamAsync(confirmDownlinkMessageBuilderResp.DownlinkPktFwdMessage);
+                    _ = SendMessageDownstreamAsync(request, confirmDownlinkMessageBuilderResp);
                 }
 
                 return new LoRaDeviceRequestProcessResult(loRaDevice, request, confirmDownlinkMessageBuilderResp.DownlinkPktFwdMessage);
+                #endregion
             }
             finally
             {
-
                 try
                 {
-                    _ = await loRaDevice.SaveChangesAsync(force: stationEuiChanged);
+                    await SaveChangesToDevice(loRaDevice, stationEuiChanged);
```

Comment:
I think this method is still async therefore should have the async suffix

Reply:
Done bdb9238b59d1c2a28d85dd1c6f06e3ddb27e440b